### ISSUE TITLE
SSL certificate for https://slack.databack.io is not valid anymore and the url does not link to a slack chat

### ISSUE THREAD

**Issue by BigAndini** [databacker/mysql-backup#193]
Hey, 

no issue on the system itself, but your ssl certificate on https://slack.databack.io/ is not valid from 7.6.2022 and it looks like the link does not link to a valid slack chat.

Regards,
Andi

**Comment by deitch** [comment]
Yeah, it is pretty old. I am switching it to the permalink invite. Should be ready in a few minutes.
